packages/@vue/cli-service/lib/commands/build/formatStats.js

```diff
@@ -22,7 +21,7 @@ module.exports = function formatStats (stats, dir, api) {
   const isMinJS = val => /\.min\.js$/.test(val)
   assets = assets
     .map(a => {
-      a.name = url.parse(a.name).pathname
+      a.name = a.name.split(`?`)[0]
```

Comment:
Please use single quotes here.
```suggestion
      a.name = a.name.split('?')[0]
```

Reply:
Sure, thank you. I just noticed that backticks are being used in other places (without any need/variables), which is the only reason why I also used them.

Reply:
So… seems we also need to fix these legacy codes sometime later. Thanks for spotting this!